### PR TITLE
[openssl] Fix pathlib

### PR DESCRIPTION
**Pull request by zabulus**
Looks like these paths were outdated, and locally I wasn't able to build it. Now it builds on all uwp triplets.

### CODE REVIEW

File: ports/openssl/CONTROL [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Source: openssl
-Version: 1.0.2k-3
+Version: 1.0.2k-4
```
Comment (codicodi) [inline ports/openssl/CONTROL]: Looks like there is a version mismatch: the one downloaded for desktop is [1.0.2l](https://github.com/Microsoft/vcpkg/blob/866dfad5ba1fbc344b51bfe7904cf38ba4a51816/ports/openssl/portfile.cmake#L7), while in CONTROL and in UWP portfile is 1.0.2k

Comment (KindDragon) [inline ports/openssl/CONTROL]: @zabulus Can you try to update UWP version to 1.0.2l?

Comment (ras0219-msft) [inline ports/openssl/CONTROL]: Unfortunately, the main OpenSSL project doesn't directly build for UWP. We're relying on the https://github.com/Microsoft/openssl/tree/OpenSSL_1_0_2_WinRT-stable-vs2017 fork to provide WinRT support, but this does mean there's a certain amount of delay.

While for most libraries we'd be fine delaying the main library update until everything is supported, `openssl` is especially sensitive to version, so we are more aggressive about updating the main project version even if that results in some skew.

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**